ee/tabby-schema/src/dao.rs

```diff
@@ -242,6 +243,7 @@ impl From<ThreadMessageAttachmentDoc> for thread::MessageAttachmentDoc {
                 thread::MessageAttachmentDoc::Issue(thread::MessageAttachmentIssueDoc {
                     title: val.title,
                     link: val.link,
+                    author: None, // will be filled in service layer
```

Comment:
i recommend remove the From trait completely (line 232 to line 264), and just use a util function in service layer to do the converstion